runtime/nvqir/cudensitymat/CuDensityMatContext.h

```diff
@@ -0,0 +1,63 @@
+/*************************************************************** -*- C++ -*- ***
+ * Copyright (c) 2022 - 2025 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#pragma once
+#include "CuDensityMatOpConverter.h"
+#include <cublas_v2.h>
+#include <cuda_runtime.h>
+#include <cudensitymat.h>
+
+namespace cudaq {
+namespace dynamics {
```

Comment:
```suggestion
namespace cudaq::dynamics {
```
reduce clutter and valid for all C++ standards we support.